click/_bashcomplete.py

```diff
@@ -68,7 +77,12 @@ def do_complete(cli, prog_name):
     except IndexError:
         incomplete = ''
 
-    for item in get_choices(cli, prog_name, args, incomplete):
+    try:
+        last_complete = cwords[cword - 1]
```

Comment:
This is the same as `args[-1]`, is this extra parameter for `get_choices` really necessary?